src/components/action-popover/action-popover.spec.js

```diff
@@ -396,15 +485,15 @@ describe("ActionPopover", () => {
         // In a normal situation the React SyntheticEvent will bubble and trigger a handler which will close the menu
         // This triggers a DOMEvent bypassing the SyntheticEvent listeners
         render();
-        const { menubutton, items } = getElements();
+        const { menubutton } = getElements();
 
         menubutton.simulate("click");
 
         act(() => {
           document.dispatchEvent(
             new CustomEvent("click", {
               detail: {
-                enzymeTestingTarget: items.first().getDOMNode(),
+                enzymeTestingTarget: menubutton.getDOMNode(),
```

Comment:
Why did we change this test, it previously use to click on the first menu item, but now it's clicking on the `menuButton`. The intention here was to test the event propagation/composedPath.